course_content/notebooks/numpy_intro.ipynb

```diff
@@ -267,9 +255,9 @@
     "\n",
     "##### Part 1\n",
     "\n",
-    "Use the code cells below to explore features of these arrays.\n",
+    "Use the code cells below to explore features of this array.\n",
     "\n",
-    "* What is the `type` of each of the arrays created above? (Hint: you can find the type of an object in Python using `type(<object>)`, where `<object>` is replaced with the name of the variable containing the object.)\n",
+    "* What is the `type` of each of the array created above? (Hint: you can find the type of an object in Python using `type(<object>)`, where `<object>` is replaced with the name of the variable containing the object.)\n",
```

Comment:
Created a grammar problem : ? should it be "of each array" ?

Reply:
Looks like it! Thanks @pp-mo